source/setup/components/ReleaseNotes.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import Dialog from "./Dialog.js";
+
+class ReleaseNotes extends Component {
+    static propTypes = {
+        onReady: PropTypes.func.isRequired,
+        releaseNotes: PropTypes.string
+    };
+
+    componentDidMount() {
+        this.props.onReady();
+    }
+
+    render() {
+        return (
+            <Dialog title={`⚡️ What's New - v${__VERSION__}`}>
```

Comment:
Can we move the condition higher so if there is no release notes it wouldn’t show an empty dialog?

Reply:
Agreed!